README.md

```diff
@@ -287,6 +306,9 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+Note that for a `TIME` column, the reader returns a `System.DateTime` value. If you need a `System.TimeSpan` column, call the
+`getTimeSpan` method in `SnowflakeDbDataReader`.
+
```

Comment:
Is there an example of this that I can add here?

Reply:
Yes, here is the example : 

`TimeSpan timeSpanTime = ((SnowflakeDbDataReader)reader).GetTimeSpan(13);`

Can you also add in that this is not a generic IDataReader method, which means that the reader must be a SnowflakeDbDataReader to be able to use this new method. This is shown in the example above where I cast the reader.

Reply:
Thanks! I added the example and mentioned that you need to cast before calling the method.